airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -139,6 +139,9 @@ def _get_log_events(self, skip: int = 0) -> Generator:
                 self.logger.warning('Error on retrieving Cloudwatch log events', error)
 
             yield from ()
+        except ConnectionClosedError as error:
```

Comment:
I think for now this is fine, but if we continue to collect more exceptions that we are just ignoring, we should perhaps flip this to just ignoring all exceptions by default and then explicitly looking for the cases where we should raise (if any).

Reply:
Could I change my PR to catch all exceptions or not?

Reply:
I think it's fine as it is for now. But if another novel case comes along, we should adjust.